igvm/commands.py

```diff
@@ -329,6 +329,8 @@ def vm_build(
         vm = es.enter_context(_get_vm(vm_hostname))
 
         if vm.dataset_obj['datacenter_type'] == 'aws.dct':
+            # check if aws_image_id is our own, if yes, skip puppet_run in cloud_init
```

Comment:
In the disaster case, IGVM should only spawn a VM as quickly as possible that has the basic config (ssh keys and puppet conf). Only this is included in the golden image.
Our failover scripts take care in the downstream steps that the packages and configs are up to date.
The AWS project will update these base images at regular intervals. We will use these golden images only in case we spawn games in AWS. All other VMs can be built with the normal Debian images from AWS, which will always include the initial puppet run.